### ISSUE TITLE
Avoid to capture message box about cookies

### ISSUE THREAD

**Issue by lducerf** [linkwarden/linkwarden#138]
Hi,
Many thanks for this awesome open source project.
Due to various regulations, some web sites display a large popup message about cookies that ides the content we want to capture.
Would it possible to hide these messages before the content is captured?
I am not a specialist at all, but maybe extensions like "I don't care about cookies" could be an inspiration source?
-> https://addons.mozilla.org/fr/firefox/addon/i-dont-care-about-cookies/
Best regards

**Comment by daniel31x13** [comment]
Thanks for pointing this out, adding it to Todo.

**Comment by boncuk10** [comment]
Hi, is there a timeline for when this will be implemented? The problem with the cookie banner is still there ...

**Comment by jfchadeyron** [comment]
+1

**Comment by xeroli** [comment]
+1

**Comment by Mat-DB** [comment]
+1

**Comment by leodbrs** [comment]
+1

**Comment by babanovac1980** [comment]
+1 on Youtube

**Comment by Choque** [comment]
Just tried out Linkwarden as an alternative to Karakeep (Hoarder) but man, most of the image and pdf captures are useless cause of some cookie banner on the website that overlays everything else. So I guess I'll stick with Karakeep for now until Linkwarden figures out how everyone else manages to not capture those banners/overlays.

**Comment by PepperPhil** [comment]
I just found this when searching for a solution where they say:
> Here is an example that closes a cookie dialog when it appears

https://playwright.dev/docs/release-notes#new-apis-2

However, I don't know where to implement it the best and if the extension to playright chromium would be a better solution:
https://chromewebstore.google.com/detail/i-dont-care-about-cookies/fihnjjcciajhdojfnbdddfaoknhalnja

**Comment by igorbaikov** [comment]
Yeah can we please get an adblock + i dont care about cookies extension?

Or use browserless/chrome which has block ads parameter out of the box.

**Comment by SawkeeReemo** [comment]
Another plus one here. It's getting worse and worse with websites lately, sadly.

**Comment by ingomc** [comment]
another plus from me

so what about a quickwin and inject hiding most common patterns via playwright?

```javascript
await page.addInitScript(() => {
  const style = document.createElement('style');
  style.textContent = `
    dialog,
    [role="dialog"],
    [role="alertdialog"],
    .cookie-banner,
    #cookie-banner,
    #cookie-consent,
    .cc-window,
    .osano-cm-dialog,
    .didomi-popup,
    .gdpr,
    .qc-cmp2-container,
    .eu-cookie-compliance,
    .cookie-consent-banner,
    .cookies,
    .dialog,
    .popup,
    .modal,
    #consent-overlay,
    .consent,
    .fc-consent-root {
      display: none !important;
    }
  `;
  document.head.appendChild(style);
});
```

I didn't look up how it is implemented, but this should be relativly quick and easy for hiding most cookie consent banner... we don't need any modal or dialog in our screenshots so hide it all

**Comment by markspercyspencer** [comment]
Any update on this? I about to drop Linkwarden because of this. I might go back to Karakeep or Raindrop
